packages/aws-lambda/src/wrapper.js

```diff
@@ -174,7 +174,13 @@ function shimmedHandler(originalHandler, originalThis, originalArgs, _config) {
       });
     };
 
-    registerTimeoutDetection(context, entrySpan);
+    /**
+     * We offer the customer to disable the timeout detection
+     * See https://github.com/instana/nodejs/pull/443
+     */
+    if (!process.env.DISABLE_LAMBDA_TIMEOUT_DETECTION || process.env.DISABLE_LAMBDA_TIMEOUT_DETECTION === 'false') {
```

Comment:
All other env vars are prefixed with `INSTANA_`. Could you prefix this one, too?

Reply:
👍

Reply:
Pushed